package.json

```
@@ -18,7 +18,7 @@
     "eslint": "^6.0.0",
     "fs-extra": "^8.0.1",
     "globby": "^9.2.0",
-    "isml-linter": "^5.27.0",
+    "isml-linter": "^5.28.1",
```

```suggestion
    "isml-linter": "latest",
```
What if we use `latest` here so that people always get the latest version?

---

I think that's a better approach, definitely